### PR TITLE
Updated OMOPMeasurementLab so the joins to the temp tables include in…

### PR DESCRIPTION
**Pull request by kembree1**
Updated OMOPMeasurementLab so the joins to the temp tables include instance_num

### CODE REVIEW

File: MSSQL/OMOPLoader.sql [hunk 1]
```diff
@@ -1768,6 +1769,7 @@ and M.encounter_num=l.encounter_num
 and M.provider_id=l.provider_id
 and M.concept_cd=l.concept_Cd
 and M.start_date=l.start_Date
+and M.instance_num = p.instance_num
```
Comment (jklann) [inline MSSQL/OMOPLoader.sql]: I think you mean l.instance_num?

**Review by jklann — CHANGES_REQUESTED** [review]
One typo I think...

File: MSSQL/OMOPLoader.sql (hunk 1 above)
Comment (kembree1) [reply MSSQL/OMOPLoader.sql]: Yep, fixed the typo

**Comment by jklann** [comment]
Add instance_num to the primary key join for measurements.

**Pull request merged**